app/controllers/hiring_staff/organisations_controller.rb

```diff
@@ -59,7 +59,7 @@ def sort_order
   end
 
   def render_draft_saved_message
-    vacancy = current_organisation.vacancies.find(params[:from_review])
+    vacancy = Vacancy.find(params[:from_review])
```

Comment:
Could this be `current_organisation.all_vacancies.find` in case of the unlikely scenario somebody knew the ID of a vacancy from another organisation?